week4/MealSharingNET/Models/Meal.cs

```diff
@@ -0,0 +1,12 @@
+namespace MealSharingNET.Models;
+public class Meal
+{
+    public int ID { get; set; }
+    public string Title { get; set; }
+    public string Description { get; set; }
+    public int Cost { get; set; }
```

Comment:
The Cost can be something like 10.19 so a more appropriate type would be decimal instead of int